spec/Section 3 -- Type System.md

```diff
@@ -1923,3 +1929,52 @@ type ExampleType {
   oldField: String @deprecated(reason: "Use `newField`.")
 }
 ```
+
+### @defer
+
+```graphql
+directive @defer(label: String, if: Boolean) on FRAGMENT_SPREAD | INLINE_FRAGMENT
+```
+
+The `@defer` directive may be provided for fragment spreads and inline fragments to 
+inform the executor to delay the execution of the current fragment to indicate 
+deprioritization of the current fragment. A query with `@defer` directive will cause
+the request to potentially return multiple responses, where non-deferred data is 
+delivered in the initial response and data deferred is delivered in a subsequent
+response. `@include` and `@skip` take precedence over `@defer`. 
+
+```graphql example
+query myQuery($shouldDefer: Boolean) {
+   user {
+     name
+     ...someFragment @defer(label: 'someLabel', if: $shouldDefer) 
+   }
+}
+fragment someFragment on User {
+  id
+  profile_picture {
+    uri
+  }
+}
+```
+
+### @stream
+
+```graphql
+directive @stream(label: String, initialCount: Int = 0, if: Boolean) on FIELD
```

Comment:
The behaviour of these arguments (and to a lesser extend the `@defer` arguments as well) is somewhat scattered across different parts of the spec. It would be nice if there was at least a brief summary of all of them here.

Reply:
I added a summary of the arguments to this section.

Reply:
Just want to note that I found the current `main` draft confusing as to whether `initialCount` is optional and appreciate the clarity in this change.